content/en/real_user_monitoring/roku/_index.md

```diff
@@ -0,0 +1,220 @@
+  tag: Github
+  text: dd-sdk-roku Source code
+- link: /real_user_monitoring
+  tag: Documentation
+  text: Explore Datadog RUM
+---
+
+{{< site-region region="gov" >}}
+
+<div class="alert alert-warning">RUM for Roku is not available on the US1-FED Datadog site.</div>
+
+{{< /site-region >}}
+
+{{< site-region region="us,us3,us5,eu,ap1" >}}
+
+<div class="alert alert-info">RUM for Roku is in beta.</div>
+
+## Overview
+
+Datadog Real User Monitoring (RUM) enables you to visualize and analyze the real-time performance and user journeys of your channel's individual users.
+
+The Datadog Roku SDK supports BrightScript channels for Roku OS 10 and higher.
+
+## Setup
+
+1. Declare the SDK as a dependency.
+2. Specify application details in Datadog.
+3. Initialize the library.
+4. Instrument the channel.
+
+### Declare the SDK as a dependency
+
+#### Using ROPM (recommended)
+
+`ROPM` is a package manager for the Roku platform (based on NPM). If you're not already using `ROPM` in your Roku project, read their [Getting started guide][1]. Once your project is set up to use `ROPM`, you can use the following command to install the Datadog dependency:
+
+```shell
+ropm install datadog-roku
+```
+
+### Setup manually
+
+If your project does not use `ROPM`, install the library manually by downloading the [Roku SDK][5] zip archive, 
+and unzipping it in your project's root folder.
+
+Make sure you have a `roku_modules/datadogroku` subfolder in both the `components` and `source` folders of your project.
+
+### Specify application details in Datadog
+
+1. Navigate to [**UX Monitoring** > **RUM Applications** > **New Application**][2].
+2. Select **Roku** as the application type and enter an application name to generate a unique Datadog application ID and client token.
+3. To disable automatic user data collection for either client IP or geolocation data, uncheck the boxes for those settings. For more information, see [RUM Roku Data Collected][9].
+
+   {{< img src="real_user_monitoring/roku/roku-new-application.png" alt="Create a RUM application for Roku in Datadog" style="width:90%;">}}
+
+To ensure the safety of your data, you must use a client token. If you used only [Datadog API keys][3] to configure the `dd-sdk-roku` library, they would be exposed client-side in the Roku channel's BrightScript code. 
+
+For more information about setting up a client token, see the [Client Token documentation][4].
+
+
+[1]: https://github.com/DataDog/dd-sdk-android
+[2]: https://app.datadoghq.com/rum/application/create
+[3]: /account_management/api-app-keys/#api-keys
+[4]: /account_management/api-app-keys/#client-tokens
+[5]: /real_user_monitoring/
+[9]: /real_user_monitoring/android/data_collected/
+
+### Initialize the library
+
+In the initialization snippet, set an environment name. For more information, see [Using Tags][8].
+
+[8]: /getting_started/tagging/using_tags/#rum--session-replay
+
+{{< /site-region >}}
+
+{{< site-region region="us" >}}
+```brightscript
```

Comment:
Do we have syntax highlighting for brightscript snippets or not? Looks like not when looking at the preview.